gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -6,15 +6,19 @@ SCL_PREFIXES = JSON.load(File.read(File.join(`git rev-parse --show-toplevel`.str
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name <%= spec.name %>
 %global plugin_name <%= spec.name.sub(/\Asmart_proxy_/, '') %>
 
 %global foreman_proxy_min_version 1.24
-%{!?_root_datadir:%global _root_datadir %{_datadir}}
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
-%global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
+%global smart_proxy_dynflow_bundlerd_dir %{!?scl:/opt/theforeman/tfm/root}%{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
I updated this in two ways:

 1) Instead of checking is this rhel, check if this is an SCL and include that beginning path
 2) Use datadir so that this can build in nonscl and SCL contexts

This is a lot of work to support non-SCL for the 1 week or less to convert to SCL :/

Reply:
I agree. Perhaps the next time we should make it clear what the goal is. Initially it wasn't clear to me we were pushing for this within 1.24 and if it isn't done in time, it'll be a blocker. We should also have a look at installer changes before we pull the trigger.

My biggest concern is that I'll be out next week and then there's 3 weeks left for me. I still need to finish the tuning option which we've prioritized. That's why I thought keeping backwards compatibility for a rollback was important.